pubspec.yaml

```diff
@@ -1,13 +1,13 @@
 name: source_gen
-version: 0.5.0
+version: 0.5.0+1
 author: Dart Team <misc@dartlang.org>
 description: Automatic sourcecode generation for Dart
 homepage: https://github.com/dart-lang/source_gen
 environment:
   sdk: '>=1.12.0 <2.0.0'
 dependencies:
-  analyzer: '^0.27.3-alpha.1'
-  build: ^0.2.1
+  analyzer: '^0.27.3'
+  build: '>=0.2.1 <0.4.0'
```

Comment:
Technically the tests wouldn't pass in 0.2.x, but I think thats ok. It does actually support that version, its just the enum values which appear in the test that would be broken.